hermes/src/network/pythnet.rs

```diff
@@ -136,11 +136,10 @@ pub async fn run(store: Arc<State>, pythnet_ws_endpoint: String) -> Result<()> {
             encoding: Some(UiAccountEncoding::Base64Zstd),
             ..Default::default()
         },
-        filters:        Some(vec![RpcFilterType::Memcmp(Memcmp {
-            offset:   0,
-            bytes:    MemcmpEncodedBytes::Bytes(b"PAS1".to_vec()),
-            encoding: None,
-        })]),
+        filters:        Some(vec![RpcFilterType::Memcmp(Memcmp::new(
+            0,                                           // offset
+            MemcmpEncodedBytes::Bytes(b"PAS1".to_vec()), // bytes
+        ))]),
```

Comment:
getting 
```
warning: use of deprecated field `solana_client::rpc_filter::Memcmp::offset`: Field will be made private in future. Please use a constructor method instead.
   --> src/network/pythnet.rs:140:13
    |
140 |             offset:   0,
    |             ^^^^^^^^^^^
    |
    = note: `#[warn(deprecated)]` on by default

warning: use of deprecated field `solana_client::rpc_filter::Memcmp::bytes`: Field will be made private in future. Please use a constructor method instead.
   --> src/network/pythnet.rs:141:13
    |
141 |             bytes:    MemcmpEncodedBytes::Bytes(b"PAS1".to_vec()),
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: use of deprecated field `solana_client::rpc_filter::Memcmp::encoding`: Field has no server-side effect. Specify encoding with `MemcmpEncodedBytes` variant instead. Field will be made private in future. Please use a constructor method instead.
   --> src/network/pythnet.rs:142:13
    |
142 |             encoding: None,
    |             ^^^^^^^^^^^^^^
    ```
    possibly due to upgrading solana packages hence the change here to fix the warnings